frontend/pages/ShoppingPage.js

```diff
@@ -20,7 +20,6 @@ export default class ShoppingPage extends React.Component {
 
     constructor(props) {
         super(props);
-        console.log(props.navigation.getParam('list'));
         this.state = {
             list: props.navigation.getParam('list')
```

Comment:
Some of the lines in this code ends with a semicolon and some doesn't.
Although automatic semicolon insertion is a feature in JavaScript, I think it is better to keep the code style consistent.

Reply:
Sure, we'll make the semi-colon style consistent